### DISCUSSION TITLE
Why Bridge. V2 not as exe?

### DISCUSSION THREAD

**Discussion by JRBros2346** [bridge-core/bridge.#417] [Q&A] [answered]
I need bridge. v2 as an app but cant find it. Why?

**Comment by KAIYOHUGO** [comment, marked as answer]
Because it use a technical call PWA. Just open bridge 2 webpage and follow the tutorial , you should can use bridge 2 offline.

**Reply by JRBros2346** [reply]
But i need internet to start the app

**Reply by solvedDev** [reply]
An internet connection is generally not required to start bridge. v2 once it's installed
